doc/project_governance/code_of_conduct.md

```diff
@@ -0,0 +1,161 @@
+
+## Questions?
+
+If you have questions, please feel free to contact the OpenTitan Code of Conduct team by emailing conduct@lowRISC.org.
+
+## Reporting Guide
+
+If you believe someone is violating the code of conduct you can always report it to the OpenTitan Code of Conduct team by emailing conduct@lowRISC.org.
+
+All reports will be kept confidential. This isn’t a public list and only members of the Code of Conduct team will receive the report.
+If you believe anyone is in physical danger, please notify appropriate law enforcement first.
+If you are unsure what law enforcement agency is appropriate, please include this in your report and we will attempt to notify them.
+If the violation occurs at an event such as a conference or workshop and requires immediate attention, you can also reach out to any of the event organisers or staff.
+Event organisers and staff will be prepared to handle the incident and able to help.
+If you cannot find one of the organisers, the venue staff can locate one for you.
+We will also post detailed contact information for specific events as part of each events’ information.
+In person reports will still be kept confidential exactly as above, but also feel free to (anonymously if needed) email conduct@lowRISC.org.
+
+## Filing a report
+
+Reports can be as formal or informal as needed for the situation at hand.
+If possible, please include as much information as you can. If you feel comfortable, please consider including:
+
+* Your contact info (so we can get in touch with you if we need to follow up).
+* Names (real, nicknames, or pseudonyms) of any individuals involved. If there were other witnesses besides you, please try to include them as well.
+* When and where the incident occurred. Please be as specific as possible.
+* Your account of what occurred. If there is a publicly available record (e.g. a mailing list archive or a public IRC logger) please include a link.
+* Any extra context you believe existed for the incident.
+* If you believe this incident is ongoing.
+* Any other information you believe we should have.
+
+## What happens after you file a report?
+
+You will receive an email from the Code of Conduct team acknowledging receipt within 24 hours (and we will aim to respond much quicker than that).
+The Code of Conduct team will immediately meet to review the incident and try to determine:
+
+* What happened and who was involved.
+* Whether this event constitutes a code of conduct violation.
+* Whether this is an ongoing situation, or if there is a threat to anyone’s physical safety.
+* Whether the incident requires escalation to the OpenTitan Governing Board
+
+If this is determined to be an ongoing incident or a threat to physical safety, the working groups’ immediate priority will be to protect everyone involved.
+This means we may delay an “official” response until we believe that the situation has ended and that everyone is physically safe.
```

Comment:
There seem to be some smart quotes in here. I'm not sure there's official guidance on them, but I haven't seen them elsewhere in the documentation.